astropy/modeling/tests/test_fitters.py

```diff
@@ -928,10 +932,10 @@ def test_2d_linear_with_weights_with_sigma_clip(self):
 
 
 @pytest.mark.skipif('not HAS_SCIPY')
-def test_fitters_with_weights():
+@pytest.mark.parametrize('fitter', [LevMarLSQFitter(), TRFLSQFitter(check_bounds=True)])
+def test_fitters_with_weights(fitter):
     """Issue #5737 """
     Xin, Yin = np.mgrid[0:21, 0:21]
```

Comment:
@krachyon this is the unit-test which seems to fail rather badly for the `TRFLSQFitter`. Note I set `check_bounds=True` which for now forces it into using the `lm` method along side the old way of enforcing parameter bounds. This essentially turns it into the `LevMarLSQFitter`.